gitpod-io/gitpod
# [perms] Implement getTeams WEB-501

## Description


- Implements getTeams
- more idiomatic permissions API
   - return boolean instead of error objects
   - don't unnecessarily expose spicedb types to application code
   - slightly more readable client-side code


## Related Issue(s)

Fixes #

## How to test

1. Start with feature flag disabled
	* Create an org
2. Enable feature flag
	* Create another org
	* You should now see both Orgs
3. Disable feature flag
	* You should see both orgs

## Documentation


#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - mp-perms-l30d62102ef</li>
	<li><b>🔗 URL</b> - <a href="https://mp-perms-l30d62102ef.preview.gitpod-dev.com/workspaces" target="_blank">mp-perms-l30d62102ef.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
</ul>

## Build Options

<details>
<summary>Build</summary>

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

</details>

<details>
<summary>Installer</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
</details>

/hold
